subworkflows/nf-core/utils_nfschema_plugin/tests/nextflow.config

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id "nf-schema@2.1.0"
+}
+
+validation {
+    parametersSchema = "${projectDir}/subworkflows/nf-core/utils_nfschema_plugin/tests/nextflow_schema.json"
```

Comment:
It's still possible to supply parameter schemas to these functions :) I'd advise against using multiple JSON schemas for one pipeline though since that complicates it a lot

Reply:
and for a meta pipeline?

Reply:
Wouldn't `workflow` be the object of the meta pipeline then? These subworkflows are also made to run at the start of a meta pipeline and not at the start of each part, right?

Reply:
Good point, I'll give in and add it as an input 😁

Reply:
Also seems like I mixed up both conversations here 😁